packages/strapi-tiptap-editor/admin/src/context/productPrice/state.tsx

```diff
@@ -20,16 +20,16 @@ const State = (props: any) => {
         },
         body: JSON.stringify({
           query: `{
-            product(id:${pageId}){
+            products(filters: { uuid: { eq: "${pageId}" } }) {
```

Comment:
Welke utility function hebben we om GraphQL queries te escapen? Zo'n functie zouden we overal moeten gebruiken.